code/modules/projectiles/projectile.dm

```diff
@@ -36,7 +36,7 @@
 	var/trajectory_ignore_forcemove = FALSE	//instructs forceMove to NOT reset our trajectory to the new location!
 
 	var/speed = 0.8			//Amount of deciseconds it takes for projectile to travel
-	var/pixel_speed = 33	//pixels per move - DO NOT FUCK WITH THIS UNLESS YOU ABSOLUTELY KNOW WHAT YOU ARE DOING OR UNEXPECTED THINGS /WILL/ HAPPEN!
+	var/pixel_speed = 32	//pixels per move - DO NOT FUCK WITH THIS UNLESS YOU ABSOLUTELY KNOW WHAT YOU ARE DOING OR UNEXPECTED THINGS /WILL/ HAPPEN!

```

Comment:
I'm going to go out on a limb and trust this comment, please either undo or justify this change.

Reply:
We have 32/32 tiles, and projectile what moves 33 pixels in step, when projectile moves 16 steps it move on 16,5 tiles, if after this it be reflected (45 degrees for test) it deal step after reflection in last move direction.
So i change step from 33 to 32 pixels.
Another way of fix is only change "setAngle()" proc to cut excess moved pixels.

Reply:
@kevinz000

Reply:
@Dennok If you tested this and it works properly I'll trust you on it, the reason I put it one pixel more was because the old code was
var/Pixel_y=round((cos(Angle)+16*cos(Angle)*2), 1)
![image](https://user-images.githubusercontent.com/2003111/40394304-be4b6c2e-5dd7-11e8-88a2-e1c4317c22cf.png)
that's what it calculated to, so I put it as 33 as I wanted to preserve projectile motion from the old system.
If your change works then so be it, the comment's there so people don't randomly change it to make projectiles go faster or slower instead of changing the speed variable, as the var's finicky. Too low and it'll sometimes stay on the same tile after multiple moves resulting in !unintended! effects, and too high and it might start clipping past tiles it's supposed to hit/hit a different tile entirely.

Reply:
Now it use vector trajectory system
https://github.com/tgstation/tgstation/blob/ca10fa6d7a70f98e6010c25b5d5451ffa0015308/code/datums/position_point_vector.dm#L133
var/speed = 32				//pixels per iteration

I tested and don't find any !unintended! effects.
Although I think that they can appear with lags, in any case.

Reply:
@Dennok Correction, you mean !!☼<<☼unintended☼>>☼!! effects that menace with spikes of bugs.